backend/src/main/java/gov/cdc/usds/simplereport/config/AuthorizationConfiguration.java

```diff
@@ -89,23 +124,75 @@
   @PreAuthorize(SPEL_HAS_PERMISSION + "READ_ARCHIVED_PATIENT_LIST" + ")")
   public @interface RequirePermissionReadArchivedPatientList {}
 
-  /** Require the current user to have the {@link UserPermission#READ_RESULT_LIST} permission. */
+  /** 
+   * Require the current user to have the {@link UserPermission#READ_RESULT_LIST} permission for
+   * the test event with UUID {@code testEventId}.
+   *
+   * <p>NOTE: any method with this annotation must have a parameter {@code testEventId}.
+   */
+  @Retention(RUNTIME)
+  @Target(METHOD)
+  @PreAuthorize(SPEL_HAS_PERMISSION + "READ_RESULT_LIST" + ")" + " && " + SPEL_CAN_VIEW_TEST_EVENT)
```

Comment:
The visually unmatched `+ ")"` bothers my mental parser.

Would almost prefer something like:
`@PreAuthorize("( " + SPEL_HAS_PERMISSION + "READ_RESULT_LIST" + ")" + " && " + SPEL_CAN_VIEW_TEST_EVENT)

Maybe it bother me because you're not doing `+ ") && " +` but breaking them out.

But it's not really an issue. And I guess not having `"(" +` keeps the lines shorter.

Reply:
The parenthesis mismatch is my fault—there's an open paren in `SPEL_HAS_PERMISSION` because it's a function call up to the argument list. I stand by it, but it does make compound expressions naaasty to parse.